pvr.octonet/addon.xml.in

```diff
@@ -1,10 +1,12 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <addon
   id="pvr.octonet"
-  version="20.3.0"
+  version="21.0.0"
   name="Digital Devices Octopus NET Client"
   provider-name="digitaldevices">
-  <requires>@ADDON_DEPENDS@</requires>
+  <requires>@ADDON_DEPENDS@
+    <import addon="inputstream.ffmpegdirect" minversion="21.0.0"/>
```

Comment:
@ksooo here is the dependency - or are we missing something else?

Reply:
No.